squeemishly/cloney-island
# Filter feature

What does this PR do?

Updates the city methods and model to fix the way the data is coming in.

Where should the reviewer start?

start with the welcome index, follow that back to the search controller, look at the city model, and the new fetch photo method on the googleplaces service.

How should this be manually tested?

http://localhost:3000/search?city=exit! should no longer work.

Any background context you want to provide?

Fixing a security flaw with the @city = eval(params[:city]) could be very harmful to our app. Please, please play around in the app before merging. There could be some blindspots that I am not seeing.

What are the relevant tickets?

#149151903

Questions:

Do Migrations Need to be ran?
rake db:migrate, rake db:seed
Do Environment Variables need to be set?
no
Any other deploy steps?
bundle
